src/Core/Constants.cs

```diff
@@ -141,6 +141,7 @@ public static class FeatureFlagKeys
     public const string NotificationBarAddLoginImprovements = "notification-bar-add-login-improvements";
     public const string AC2476_DeprecateStripeSourcesAPI = "AC-2476-deprecate-stripe-sources-api";
     public const string PersistPopupView = "persist-popup-view";
+    public const string EnableNewCardCombinedExpiryAutofill = "enable-new-card-combined-expiry-autofill",
```

Comment:
@jprusik Checkout the github actions errors here. This isn't formatted correctly.

Reply:
ah yeah; copy-paste will get you every time. Fixed; thanks 👍